ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -59,14 +72,41 @@ function BlockaidBannerAlert({ txData, ...props }) {
 
   const t = useContext(I18nContext);
   const { updateTransactionEventFragment } = useTransactionEventFragment();
+  const [isLoadingDisplayed, setIsLoadingDisplayed] = useState(false);
+
+  useEffect(() => {
+    if (securityAlertResponse?.reason === 'loading') {
+      setIsLoadingDisplayed(true);
+    }
+  }, [securityAlertResponse]);
 
   if (
     !securityAlertResponse ||
     Object.keys(securityAlertResponse).length === 0
   ) {
     return null;
   } else if (securityAlertResponse.reason === 'loading') {
-    return <LoadingIndicator isLoading />;
+    return (
+      <Box
+        alignItems={AlignItems.center}
+        borderColor={BorderColor.borderMuted}
+        borderRadius={BorderRadius.SM}
+        className="blockaid_banner__wrapper"
+        display={Display.Flex}
+        margin={props.margin}
+        marginLeft={props.marginLeft}
+        marginRight={props.marginRight}
+        padding={2}
+      >
+        <Spinner
+          className="blockaid_banner__spinner"
+          color={BorderColor.borderMuted}
```

Comment:
```suggestion
          color="var(--color-icon-default)"
```
the value wasn't translating right when used for the SVG fill attributes

#### before:
<img width="484" alt="Screenshot 2024-02-21 at 10 23 48 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/a1de8672-be0a-416a-9c95-b221b9c990b3">
<img width="1434" alt="Screenshot 2024-02-21 at 10 27 36 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/3d99eb04-592c-4c06-b04f-8c0258517a11">


#### after:
<img width="617" alt="Screenshot 2024-02-21 at 10 35 37 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/16c9c123-85c0-4bfc-b8fa-2ed099fdcd36">